ci/tests/arakoon/extend_promote_demote_test.py

```diff
@@ -287,6 +275,49 @@ def is_master_node(node_ip):
     assert len(incorrect_configs) == 0, 'Incorrect arakoon config contents: \n{0}'.format('\n'.join(incorrect_configs))
 
 
+def validate_arakoon_config_content(config_file, node_ids):
+    ips = node_ids.keys()
+    ips.sort()
+
+    first_ip = ips[0]
+    client = SSHClient(first_ip, username='root')
+    contents = client.run('etcdctl get {0}'.format(config_file))
+    cfg = RawConfigParser()
+    cfg.readfp(StringIO(contents))
+
+    logger.info('Arakoon config to validate:\n{0}'.format(str(contents)))
+
+    cluster_id_from_filename = config_file.split('/')[-2]
+    assert cfg.has_section('global'), 'Arakoon config {0} has no global section'.format(config_file)
+    assert cfg.has_option('global', 'cluster'), 'Arakoon config {0} has no option cluster in global section'.format(config_file)
+
+    cluster = list()
+    for entry in cfg.get('global', 'cluster').split(','):
+        cluster.append(entry)
+
+    logger.info('cluster: {0}'.format(cluster))
+    logger.info('node_ids: {0}'.format(node_ids))
+    for node_id in node_ids.values():
```

Comment:
Merge these loops
for node_id in node_ids.values()
for cluster_ip, cluster_id in node_ids.iteritems()